### ISSUE TITLE
False positive on MagicHash

### ISSUE THREAD

**Issue by bergmark** [ndmitchell/hlint#405]
Source
```
{-# LANGUAGE MagicHash #-}

module Foo where

foo _ = "_id"#
```

hlint:
```
Foo.hs:1:1: Warning: Unused LANGUAGE pragma
Found:
  {-# LANGUAGE MagicHash #-}
Why not remove it.
```

Fixing hint:
```
Foo.hs:4:1: error:
    parse error (possibly incorrect indentation or mismatched brackets)
```

Seems to have been introduced in #358, ping @mitchellwrosen

**Comment by mitchellwrosen** [comment]
Here ya go #406

**Comment by bergmark** [comment]
Thanks!
